### DISCUSSION TITLE
Pause server.

### DISCUSSION THREAD

**Discussion by starrider346** [pelican-dev/panel#114] [Ideas]
Docker has to ability to pause a server and resume it's processes later on. Would that be possible to integrate into the panel?

**Comment by Boy132** [comment]
This could be potentially dangerous because not every server supports it, so they would crash on pause/ resume.

**Comment by parkervcp** [comment]
I haven't looked at what "pausing" in docker does but if it can maintain the memory of a software it "might" be viable.

The issue I have with this is that we would have no way to detect when to resume a server.

**Comment by lancepioch** [comment]
We will consider adding this to the backend API, but I doubt we will add it to the UI.
